sql/src/main/java/io/crate/expression/symbol/Symbols.java

```diff
@@ -105,6 +114,16 @@ public static void toStream(Symbol symbol, StreamOutput out) throws IOException
         return in.readList(Symbols::fromStream);
     }
 
+    @Nullable
+    public static Symbol nullableFromStream(StreamInput in) throws IOException {
+        int ordinal = in.readVInt();
```

Comment:
`readVInt` uses 5 bytes for negative values, so maybe it would be better to write/read a Boolean to indicate the nullability. Given that the  ordinals are small numbers, we'd have 2 bytes for most cases.